api/api.js

```diff
@@ -19,14 +19,16 @@ var debug = require('../tools/debug');
 var routes = require('./routes.js');
 
 var PRIVATE = require('../PRIVATE.json');
-var maestro = require('./maestro')(PRIVATE.token); // API-side mqtt client
+
+// Creating API-side MQTT client: maestro !
+var maestro = require('./maestro')(PRIVATE.token);
 
 var server = new http.Server(app);
 var io = require('socket.io')(server);
 
-
 var PORT = 4000;
 
+var sockets = [];
```

Comment:
Ok, i let this `sockets` here because i didn't really know what to do with it, but if `io.sockets.emit()` does the job, then let's just dump it, right ?